### PR TITLE
Auto build/publish docker camera server

### PR DESCRIPTION
**Pull request by danjampro**
The goal is to make github automatically build and publish the camera server docker image to docker hub (huntsmanarray/camera_server:latest) following any push to the develop branch. 

After some trial and error, this now works. Docker buildx is currently necessary to build the ARM image using github workflows.

### CODE REVIEW

File: .github/workflows/camera_server.yml [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
-    - uses: actions/checkout@v1
-    - name: docker_build_cameraserver
-      run: docker build . --file docker/camera_server.docker --tag camera_server:$(date +%s)
+      -
+        name: Checkout
+        uses: actions/checkout@v1
+      -
+        name: Set up Docker Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
```
Comment (wtgee) [inline .github/workflows/camera_server.yml:18]: `crazy-max` eh? :)

File: docker/camera_server.docker [hunk 2]
```diff
@@ -73,16 +73,18 @@ RUN pip install pymongo
...
-
 #Mount the SSHFS & start pyro camera server (blocking)
 CMD python $HUNTSMAN_POCS/scripts/pyro_camera_server.py 
 
 
+
+
+
+
+
```
Comment (wtgee) [inline docker/camera_server.docker]: Not terribly important but also no reason to have all these blank lines.

**Review by wtgee — APPROVED** [review]
I haven't really looked into the github actions enough, but sounds like it is working and building so LGTM!

File: .github/workflows/camera_server.yml (hunk 1 above)
Comment (danjampro) [reply .github/workflows/camera_server.yml:18]: 100% legit

**Pull request merged**